sdk/notificationhubs/notification-hubs/src/models/registration.ts

```diff
@@ -458,21 +396,6 @@ export interface MpnsRegistrationDescription extends RegistrationDescriptionComm
   type: "Mpns";
 }
 
-/**
```

Comment:
I wonder whether there's still windows phones out there.  Should we remove everything Mpns/Windows Phone related?

Reply:
There could still be MPNS registrations in the customers databases, even if they are no longer used.  That's why that is kept here even though it shouldn't be used for targeting, installations, etc.